doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -49,32 +55,36 @@ Print a usage message.
 
 =item B<-module> I<filename>
 
-Filename of a fips module to perform an integrity check on.
+Filename of the FIPS module to perform an integrity check on.
```

Comment:
The flags say module, so I want to keep the term module.  I think "FIPS module" is a commonly-known term, and "crypto provider" or "FIPS provider" is more openssl-ish.  Okay?

Reply:
I don't understand why you changed "a fips" to "the FIPS". If I remember correctly, this command is public so that others may use it for their own module. If it's only for ourselves, there's no reason whatsoever to make it a public command.

Reply:
The intro section several times says "a fips module"  In this section, the -module flag identifies the module to use.  I think it's okay.